en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md

```diff
@@ -0,0 +1,39 @@
+# Taking Memory Dumps
```

Comment:
The title and file name/left-navigation name don't seem to match very well. Debugging memory and taking a memory dump are like different things.